style.css

```diff
@@ -29,3 +29,28 @@ This theme, like WordPress, is licensed under the GPL.
     /* add overflow rule */
     overflow: hidden;
 }
+/*override wp font size css*/
+.has-small-font-size {
+    font-size: 0.8rem;
+}
+.has-body-normal-font-size {
+	font-size: 1rem;
```

Comment:
That we are using `1rem` here may indicate this pull request is unnecessary.

Reply:
yes very correct. the normal size on Wordpress is the same as Vocabulary's body normal which is the base line font size. I will proceed to close this PR.